plugins/mcreator-core/procedures/entity_get_shootpower.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
> You don't need to request projectile type here, just cast to IProjectile.

Remove projectile type here due to this reason.

Reply:
Done! Last commit changed it.